src/riak_cs_riak_client.erl

```diff
@@ -331,6 +331,8 @@ get_user_with_pbc(MasterPbc, Key, true) ->
 get_user_with_pbc(MasterPbc, Key, false) ->
     case strong_get_user_with_pbc(MasterPbc, Key) of
         {ok, _} = OK -> OK;
+        {error, <<"{pr_val_unsatisfied,", _/binary>>} ->
```

Comment:
How about matching fully with `{pr_val_unsatisfied,3,2}` ? `3,2` should be negligible but could `3,1` happen in faulty cluster and that's the case we want to warn operators?